scripts/mkcloud

```diff
@@ -475,15 +483,22 @@ function setuphost()
             ipxe=qemu-ipxe
         }
         [[ $arch = s390x ]] && kvmpkg=qemu-s390
-        zypper --non-interactive in --no-recommends \
-            libvirt $kvmpkg $osloader $ipxe lvm2 curl wget bridge-utils \
-            dnsmasq netcat-openbsd ebtables libvirt-python
-        [ "$?" == 0 -o "$?" == 4 ] || complain 10 "setuphost failed to install required packages"
+
+        ensure_packages_installed libvirt $kvmpkg $osloader $ipxe \
+            lvm2 curl wget bridge-utils dnsmasq netcat-openbsd \
+            ebtables libvirt-python
+
+        # enable UEFI boot
+        [[ $arch = x86_64 ]] && [[ -n $want_efi ]] && {
+            zypper --non-interactive ar http://download.suse.de/ibs/Devel:/Virt:/SLE-12-SP2/SUSE_SLE-12-SP2_GA_standard/Devel:Virt:SLE-12-SP2.repo
```

Comment:
that would be wrong on SP1 hosts (e.g. cloud 6.x). I think easiest is to make that a cloud7 only feature and then we can remove this hack